docs/documentation/release_notes/topics/26_1_0.adoc

```diff
@@ -43,6 +49,39 @@ ifeval::[{project_community}==true]
 Many thanks to https://github.com/sonOfRa[Simon Levermann] for the contribution.
 endif::[]
 
+= Support for prompt=create
+
+There is a support for the https://openid.net/specs/openid-connect-prompt-create-1_0.html[Initiating user registration standard], which allows OIDC client to initiate the login request with
+the parameter `prompt=create` to notify {project_name} that new user should be registered rather than existing user authenticated. Initiating user registration was already supported in {project_name} with the use of dedicated endpoint `/realms/<realm>/protocol/openid-connect/registrations`
+however this endpoint is now deprecated in favor of the standard way as it was proprietary solution specific to {project_name}.
```

Comment:
```suggestion
Support now exists for the https://openid.net/specs/openid-connect-prompt-create-1_0.html[Initiating user registration standard], which allows OIDC clients to initiate the login request with
the parameter `prompt=create` to notify {project_name} that new user should be registered rather than the existing authenticated user. Initiating user registration was already supported in {project_name} with the use of dedicated endpoint `/realms/<realm>/protocol/openid-connect/registrations`,
however this endpoint is now deprecated in favor of the standard way as it was proprietary solution specific to {project_name}.
```